src/inmanta/server/services/resourceservice.py

```diff
@@ -936,7 +942,8 @@ async def update_fields_resource(
                     # transient resource update
                     if not is_resource_action_finished:
```

Comment:
the list of resource ids for the resource action is not filtered 

I find this hard to review, I would simply filter the list of resources and resource_ids higher up in the code, so the rest of the code is more uniform